src/styles/index.scss

```diff
@@ -3 +1,54 @@
+@import './abstract/constants';
+@import './abstract/placeholders';
+@import './base/base';
+
+.calendar {
+  display: flex;
+  padding: $calendar-indent;
+  flex-wrap: wrap;
+  align-items: center;
+  gap: $gap;
+
+  @extend %calendar-width;
+
+  @each $day, $index in $days-of-week {
+    &--start-day-#{$day} :first-child {
+      margin-left: ($day-size + $gap) * $index;
```

Comment:
The margin-left calculation seems to be wrong. It should be ($day-size + $gap) * ($index - 1); because $index starts from 1 and we need to skip the first day. (Bug)

Reply:
The index start from 0